### DISCUSSION TITLE
whats the difference between QR codes generated on github versus on the app?

### DISCUSSION THREAD

**Discussion by defcomedyjam** [gopro/labs#591] [Q&A] [answered]
both used the same commands !MBYPS=1!MDLRV=1!MLEVL=9!MEXPS=1
but the QR codes look different somehow.

github
![2023-10-08_092313](https://github.com/gopro/labs/assets/123236393/da16e5eb-fc8d-4355-a5c6-11b85d9d9bd2)
app
![Screenshot_20231008_092128](https://github.com/gopro/labs/assets/123236393/958b34b9-ab3d-4281-9035-c59c01ba89fb)

**Comment by dnewman-gpsw** [comment, marked as answer]
Nothing much, same info is encoded.  Seems the web defaults to using more error correction, resulting in the more complex image.
